src/databricks/labs/ucx/hive_metastore/workflows.py

```diff
@@ -73,6 +73,13 @@ def refresh_migration_status(self, ctx: RuntimeContext):
         """Refresh the migration status to present it in the dashboard."""
         ctx.tables_migrator.index_full_refresh()
 
+    @job_task(job_cluster="table_migration", depends_on=[migrate_external_tables_sync, migrate_dbfs_root_delta_tables,
+                                                         migrate_dbfs_root_non_delta_tables, migrate_views, refresh_migration_status])
+    def refresh_not_migrated_status(self, ctx: RuntimeContext):
+        """Refresh the not migrated tables status to present it in the dashboard."""
+        ctx.tables_migrator.get_remaining_tables(
+            # workspace_name=ctx.workspace_info.current()
+        )
```

Comment:
When I pass `workspace_name=ctx.workspace_info.current()` here, I get an error in the task run, and the task fails. Ideally, I wanted to add the workspace_name to the logs as well - for now, it might be unnecessary.

Reply:
Can you share the error?

Reply:
This is the error I get:

Reply:
If you do not get this to work, no worries, let's skip it for now, it is not too important. Another route you could try:

``` python
ctx.workspace_client.get_workspace_id()
```